rfcs/text/0014-extend-file-pe.md

```diff
@@ -77,17 +78,217 @@ Stage 1: Provide a high-level description of example sources of data. This does
 
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+https[://]c4a08b8258a848efb518dcea18a1d11e[.]us-central1[.]gcp[.]cloud[.]es[.]io/goto/51d883b52d4ad99d9200c1b25cf1b5c9
 -->
-
+```
+"file": {
+  "pe": {
+    "machine_type": 332,
+    "flattened": {
```

Comment:
Based on the definitions [here](https://github.com/elastic/ecs/blob/master/rfcs/text/0014/pe.yml), both `debug` and `resources` would be directly underneath the `pe.*` top-level?

Reply:
@ebeahan do you mean that in `0014-extend-file-pe.md` there's this example
```
"file": {
  "pe": {
    "machine_type": 332,
    "flattened": {
      "debug": [
        {
...
```
and in `pe.yml` it's reflected like this
```
---
- name: pe

  fields:

    - name: icon.hash.dhash
      level: extended
      type: keyword
      description: >
        Difference Hash (dhash) to find files with a visually similar icon or thumbnail.

      example: b806e17c8e330d82

    - name: debug
      level: extended
      type: nested
      description: >
        Debug information, if present
...
```

@dcode, was the intention to be `file.pe.{debug,resources}.*` or `file.pe.flattened.{debug,resources}.*`

Reply:
Questions caused be stale data examples. Refactoring.